app/posts/apply-for-teacher-training/2020-01-16-question-protocol-for-pilot.md

```diff
@@ -0,0 +1,1722 @@
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Providers ask for documentation"],
+    notes: ["In addition to status, do we also need to ask for the date they entered the country? This might be relevant if an applicant has a Tier 5 Visa (from Australia or New Zealand, where people can work for up to 2 years in the UK). This may also impact funding decisions"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Main language spoken",
+  items: {
+    question: "Is English your main language?",
+    required: "Yes",
+    api: {
+      param: "candidate.english_main_language",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#candidate-object"
+    },
+    why: [{
+      who: ["unknown"],
+      what: "Further research required"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    notes: ["Does having an English language GCSE (or equivalent) mean we need to ask this question? Does an English GCSE indicate fluency?"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "English language skills/qualifications",
+  items: {
+    question: "Please tell us about your English language qualifications (including grades or scores)",
+    required: "Yes (if candidate’s main language is not English)",
+    api: {
+      param: "candidate.english_language_qualifications",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#candidate-object"
+    },
+    why: [{
+      who: ["unknown"],
+      what: "Further research required"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    notes: ["Does having an English language GCSE (or equivalent) mean we need to ask this question? Does an English GCSE indicate fluency?"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Other languages spoken",
+  items: {
+    question: "If you are bilingual or very familiar with languages other than English, you can tell us about them here",
+    required: "No",
+    api: {
+      param: "candidate.other_languages",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#candidate-object"
+    },
+    why: [{
+      who: ["candidate"],
+      what: "may want to demonstrate language skills, especially if looking to teach a language subject or teach in school with children whose main language isn’t English"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Assesed at interview"],
+    notes: ["SCITTs/SDs we've recently spoken to haven’t expressed a need for this, but it’s something candidates sometimes like to mention"]
```

Comment:
Something about being pertinent to modern languages courses?

Reply:
That’s included in the who/what part of the protocol. Have added ‘modern’.